test/Makefile

```diff
@@ -271,6 +271,11 @@ generate-mocks: ## Generates mocks for the tests, using mockery tool
 	mockery --name=stateInterface --dir=../synchronizer --output=../synchronizer --outpkg=synchronizer --structname=stateMock --filename=mock_state.go
 	mockery --name=Tx --srcpkg=github.com/jackc/pgx/v4 --output=../synchronizer --outpkg=synchronizer --structname=dbTxMock --filename=mock_dbtx.go
 
+	mockery --name=EtherscanI --dir=etherman/etherscan --output=etherman --outpkg=etherman --structname=etherscanMock --filename=mock_etherscan.go
+	mockery --name=EthGasStationI --dir=etherman/ethgasstation --output=etherman --outpkg=etherman --structname=ethGasStationMock --filename=mock_ethgasstation.go
+	mockery --name=HttpI --dir=etherman/restclient --output=etherman/etherscan --outpkg=etherscan --structname=httpMock --filename=mock_http.go
+	mockery --name=HttpI --dir=etherman/restclient --output=etherman/ethgasstation --outpkg=ethgasstation --structname=httpMock --filename=mock_http.go
+
```

Comment:
this should also change if you accept my suggestion regarding the interfaces and http abstraction